src/express-tooling.ts

```diff
@@ -58,14 +58,15 @@ function requireScopesMiddleware(scopes: string[],
           validateScopes(req, res, next, scopes, logger);
         }
       })
-      .catch(err => {
+      .catch(error => {
         try {
-          precedenceErrorHandler(err, logger);
-        } catch (err) {
-          logOrNothing.error(`Error while executing precedenceErrorHandler: ${err}`, logger);
+          precedenceErrorHandler(error, logger);
```

Comment:
or another chained catch block ;)

Reply:
👍  I try that

Reply:
I am not sure about that actually, because this would be a new pattern for us.